app/src/main/kotlin/io/sweers/catchup/ui/fragments/service/ServiceFragment.kt

```diff
@@ -312,6 +316,13 @@ class ServiceFragment : InjectingBaseFragment(),
         recyclerView.show()
         recyclerView.itemAnimator = defaultItemAnimator
       }
+      detailDisplayed = getBoolean("detailDisplayed")
+
+      if (detailDisplayed) {
+        // This is necessary to support state restoration in IRV, which expects the page to be
+        // bound after rotation before restoration.
+        detailDisplayer.bindOnly(recyclerView)
```

Comment:
Do you also need to unbind when this fragment is destroyed?

Reply:
For right now - no, as the only way this fragment gets destroyed is if it's collapsed. If there's ever a master->detail type flow for tablets though, then maybe!

Reply:
Cool cool